tests/test_schema.py

```diff
@@ -30,13 +30,57 @@ def load_json(filename):
     return doc
 
 
+def test_delivery():
+    """Test delivery superset schema."""
+    upsert = load_json('valid.json')
+    takedown = copy.deepcopy(upsert)
+    takedown['action'] = 'takedown'
+    assert schema.validate_schema(schema.delivery, upsert) == upsert
+    assert schema.validate_schema(schema.delivery, takedown) == takedown
+
+
 def test_empty_document():
     """Test that an empty document doesn't validate."""
     doc = load_json('empty.json')
     with pytest.raises(schema.MultipleInvalid):
         schema.track_bundle(doc)
 
 
+@pytest.mark.parametrize('type_', (
+    'AdvertsiementSupportedModel',
+    'DevicefeeModel',
+    'PayAsUGoModel',
+    'Rights',
+    '$5',
+))
+def test_invalid_commercial_model_type(type_):
+    with pytest.raises(schema.CommercialModelTypeInvalid):
+        assert schema.CommercialModelType(type_) == type_
+
+
+@pytest.mark.parametrize('type_', (
+    ['ConditionalDownload', 'IntStream',],
```

Comment:
1. Lists don't need trailing commas (values spanning multiple lines should include them, but on a single line you don't need them).